doc/user/content/sql/create-cluster-replica.md

```diff
@@ -47,6 +47,11 @@ _replica_name_ | A name for this replica.
 
 {{% replica-options %}}
 
+{{< note >}}
+If you do not specify an availability zone,  Materialize should automatically assign the availability zone with the least existing replicas. To view all replica availability zones, use
+the [`show mz_cluster_replicas`](/docs/sql/system-catalog/mz_catalog/#mz_cluster_replicas) command.
```

Comment:
(also applies for the instance below)

The [`mz_cluster_replicas`](https://materialize.com/docs/sql/system-catalog/mz_catalog/#mz_cluster_replicas) is a catalog table, not a command, so users should type:

```sql
select * from mz_cluster_replicas
```

(at least until we extend the `show cluster replicas` command to show the availability zone). cc @ggnall to decide whether we should do this.

Reply:
I see, I did not clarify from the Slack thread. I'll update and rerequest reviews.